include/ck/utility/generic_memory_space_atomic_add.hpp

```diff
@@ -4,22 +4,7 @@
 namespace ck {
 
 template <typename X>
-__device__ X atomic_add(X* p_dst, const X& x);
```

Comment:
Please don't remove this. This template declaration (but without definition) is intentional.

Reply:
Fix in 220985c9c54594f0d9ae4f73ea2dc5f8eb56e9b5